### PR TITLE
Reword some sentences from #948

### PR DESCRIPTION
**Pull request by ann0see**
**Short description of changes**
Fixes some errors found in #948 

**Context: Fixes an issue? Related issues**
NO

**Status of this Pull Request**
Ready for review

**What is missing until this pull request can be merged?**
Nothing

**Does this need translation?**
YES

## Checklist

- [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/main/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
- [x] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
- [x] I'm sure that this Pull Request goes to the correct branch

### CODE REVIEW

File: wiki/en/FAQ.md [hunk 1]
```diff
@@ -68,6 +68,6 @@ Adding video support adds a lot of complexity. You can use other software like J
...
 
-Normally, network address translation (NAT) firewalls prevent incoming requests initiated from outside the local network. Inbound traffic is only possible for packets relating to an outbound request (strictly speaking, “related” and “established” packets to an initial outbound connection). In Registered Server mode, when your Jamulus Server connects to a Directory it of course initiates an outbound connection. From then on, the Directory sends (established/related) connection “pings” (not ICMP pings) to your Server at regular intervals to keep the relevant NAT port(s) open on your router/firewall.
+Normally, network address translation (NAT) prevents incoming requests initiated from outside the local network. Inbound traffic is only possible for packets relating to an outbound request (strictly speaking, “related” and “established” packets to an initial outbound connection). In Registered Server mode, when your Jamulus Server connects to a Directory it of course initiates an outbound connection. From then on, the Directory sends (established/related) connection “pings” (not ICMP pings) to your Server at regular intervals to keep the relevant port(s) open on your router.
```
Comment (mcfnord) [inline wiki/en/FAQ.md]:
```suggestion
Normally, network address translation (NAT) blocks incoming requests initiated from outside the local network. Inbound traffic is only possible for packets relating to an outbound request (strictly speaking, “related” and “established” packets to an initial outbound connection). In Registered Server mode, your Jamulus Server initiates an outbound connection when it connects to a Directory. From then on, the Directory sends (established/related) connection “pings” (not ICMP pings) to your Server at regular intervals to keep the relevant port(s) open on your router.
```

File: wiki/en/Unregistered-Servers.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ Normally, people from outside your home network cannot see things inside it. To
 
 ### Getting the public IP
 
-To allow others to connect to your Server from the internet, give them your public internet IP address. You can [learn your current address using Google](https://www.google.com/search?q=what+is+my+ip).
+To allow others to connect to your Server from the internet, give them your public IP address. You can [learn your current address using Google](https://www.google.com/search?q=what+is+my+ip).
```
Comment (gilgongo) [inline wiki/en/Unregistered-Servers.md]: Slightly ambiguous if someone on  LAN thinks that public is anything that's not localhost, but maybe it's not a problem,

File: wiki/en/Tips-Tricks-More.md [hunk 3]
```diff
@@ -102,7 +102,7 @@ Make sure you connect your MIDI device's output port to the Jamulus MIDI in port
 
 ### Converting a Registered Server to an Unregistered one on the fly
 
-You can run as a Registered Server long enough for your band to connect, then go "private" (Unregistered) by setting the Directory to "none" in the Server GUI. Your band mates will still be connected to the Server until they disconnect. (Thanks to [David Savinkoff](https://github.com/DavidSavinkoff) for this tip!)
+You can run as a Registered Server long enough for your group to connect, then go "private" (Unregistered) by setting the Directory to "none" in the Server GUI. Musicians will still be connected to the Server until they disconnect. (Thanks to [David Savinkoff](https://github.com/DavidSavinkoff) for this tip!)
```
Comment (gilgongo) [inline wiki/en/Tips-Tricks-More.md]: The style guide would prefer "long enough for people to connect", I think.

File: wiki/en/Unregistered-Servers.md (hunk 2 above)
Comment (ann0see) [reply wiki/en/Unregistered-Servers.md]: I don't think that's a problem as long as we direct them to the Google link where they'd find their IP.

File: wiki/en/Tips-Tricks-More.md (hunk 3 above)
Comment (ann0see) [reply wiki/en/Tips-Tricks-More.md]: Ah that's better.

File: wiki/en/Unregistered-Servers.md (hunk 2 above)
Comment (pljones) [reply wiki/en/Unregistered-Servers.md]: Perhaps, then, for greater clarity, change
> learn your current address

to
> learn your current public IP address

as that's what's being referred to.

Comment (ann0see) [reply wiki/en/Unregistered-Servers.md]: Done.

File: wiki/en/Tips-Tricks-More.md (hunk 3 above)
Comment (ann0see) [reply wiki/en/Tips-Tricks-More.md]: Done.

**Review by pljones — APPROVED** [review]

**Pull request merged**